scripts/run_e2e_tests.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -x
+
+E2E_TEST_NS=gitops-test
+E2E_TEST_DIR=./test/e2e
+
+
+echo "Checking if operator-sdk is installed"
+if ! command -v operator-sdk &> /dev/null
+then
+    echo "Unable to find operator-sdk"
+    exit
```

Comment:
Should we check for the version of the operator-sdk?

Are we compatible with all versions?

Reply:
Updated the script to check the version. We should consider updating operator-sdk in a different branch. The latest version is 1.2.0